tests/utils.py

```diff
@@ -399,7 +398,7 @@ def post_send_ex(agr_obj, send_object, gid_index, port, send_op=None, qp_idx=0):
     qp.wr_complete()
 
 
-def post_send(agr_obj, send_wr, gid_index, port, qp_idx=0):
+def post_send(agr_obj, send_wr, gid_index, port, qp_idx=0, ah=None):
```

Comment:
remove gid_index and port as they're not in use anymore (callers should be recursively updated)

Reply:
Ack